src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,139 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationServiceImpl service;
+    private static User user1;
+    private static User user2;
+    private static User user3;
+    private static final long DEFAULT_ID = 200L;
+    private static final String DEFAULT_LOGIN = "sasaylalita";
+    private static final String DEFAULT_PASSWORD = "qwerty12345";
+    private static final int DEFAULT_AGE = 28;
+
+    @BeforeAll
+    static void beforeAll() {
+        service = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        user1 = new User();
+        user1.setId(DEFAULT_ID);
+        user1.setLogin(DEFAULT_LOGIN);
+        user1.setPassword(DEFAULT_PASSWORD);
+        user1.setAge(DEFAULT_AGE);
```

Comment:
To reduce the amount of the lines of code you could create a constructor in class `User`.

Also, here you can see a repetitive duplicated logic. It can be extracted into the separate method.